### ISSUE TITLE
Thumbnail: This video element does not have a track with kind=captions.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3828]
Description: This video element does not have a track with kind=captions.

Code Snippet: 

<video class="ng-star-inserted" src="https://www.learningcontainer.com/wp-content/uploads/2020/05/sample-mp4-file.mp4" controls="">Failed to load http://picsum.photos/id/1008/400</video>

Expected: Developers must ensure that multimedia files provide the equivalent synchronized alternatives for all synchronized media or multimedia presentations. The method for providing synchronized alternatives varies based on media types. Ensure open or closed captioning is used for any presentations containing audio and video.
Captions need to include:
•	All dialog
•	Important non-dialog audio content including sounds and music which are necessary for understanding
•	Identification of the speaker, if more than one speaker or person is present
Quality captions need to be:
•	Accurate
•	Consistent
•	Clear
•	Readable
•	Equal
See http://www.captioningkey.org/
For videos with a purely musical audio track, the best practice is to put a single [MUSIC PLAYING] caption frame that drops off-screen after 10 seconds or so. The same would be true of a completely silent file - using a [NO SPEECH] tag or something similar. In order to create an equal experience for Deaf/Hard-of-Hearing viewers, it is important to articulate the auditory experience - even if there is no auditory experience (otherwise, they might think they're missing something).

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98774124-5e979080-2410-11eb-88ab-1c24396a7445.png)

**Comment by manu-kr** [comment]
Changes are done. need to push the changes.

**Comment by puru-hk** [comment]
Tested it is working as expected.
